installations/settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':app'
+rootProject.name = "My Application"
```

Comment:
Don't forget to add this project to the root settings.gradle and the CI scripts.

Reply:
Done

Reply:
Hi

Reply:
Hello